email-manager/src/main.rs

```diff
@@ -60,11 +58,8 @@ fn main() -> Result<()> {
 
     // Check completion command BEFORE entities and services initialization.
     // Related issue: https://github.com/soywod/himalaya/issues/115.
-    match compl_arg::matches(&m)? {
-        Some(compl_arg::Command::Generate(shell)) => {
-            return compl_handler::generate(create_app(), shell);
-        }
-        _ => (),
+    if let Some(compl_arg::Command::Generate(shell)) = compl_arg::matches(&m)? {
```

Comment:
I would prefer to keep the original version in case we add more commands in the future (plus to match the other matches style).

Reply:
Added a `#[allow(clippy::single_match)]` above the main function.